app/code/Magento/Catalog/Test/Mftf/Test/AdminAddImageToWYSIWYGProductTest.xml

```diff
@@ -75,6 +75,7 @@
         <waitForLoadingMaskToDisappear stepKey="waitForLoading13"/>
         <see selector="{{ProductShortDescriptionWYSIWYGToolbarSection.CreateFolder}}" userInput="Create Folder" stepKey="seeCreateFolderBtn2" />
         <waitForLoadingMaskToDisappear stepKey="waitForLoading14"/>
+        <wait stepKey="wait" time="3"/>
```

Comment:
Why we need it? and why it wasn't needed earlier?

Reply:
The test fails since there’s a delay so I added it.